### PR TITLE
add e11s ja translation

### PR DESCRIPTION
**Pull request by r4ha**
@MaikoTan review this please!

### CODE REVIEW

File: ui/raidboss/data/05-shb/raid/e11s.js [hunk 1]
```diff
@@ -130,6 +130,7 @@ export default {
           en: 'Protean -> Partner Stacks',
           de: 'Himmelsrichtung -> Auf Partner sammeln',
           fr: 'Position -> Packez-vous avec votre partenaire',
+          ja: '8方向散 -> 2人頭割り',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/raid/e11s.js:133]: '8方向散開'

And I prefer 'ペア頭割り', what do you think?

File: ui/raidboss/data/05-shb/raid/e11s.js [hunk 2]
```diff
@@ -275,20 +281,23 @@ export default {
...
             fr: 'Packez-vous avec le lien de Feu',
+            ja: '炎の線を頭割り',
             ko: '화염 선 대상자, 쉐어뎀',
             cn: '和火连线分摊',
           },
           lightningTetherOnYou: {
             en: 'Take Lightning To Tanks',
             de: 'Bring Blitz zum Tank',
             fr: 'Donnez l\'Éclair aux tanks',
+            ja: 'タンク側へ',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/raid/e11s.js:292]: What about 'タンクに近づく'?

**Review by MaikoTan — COMMENTED** [review]
few minor suggestions

**Comment by r4ha** [comment]
done.

**Pull request merged**